Tests/TuistKitTests/Services/Project/ProjectListServiceTests.swift

```diff
@@ -35,39 +36,43 @@ final class ProjectListServiceTests: TuistUnitTestCase {
     }
 
     func test_project_list() async throws {
-        // Given
-        given(listProjectsService)
-            .listProjects(serverURL: .value(serverURL))
-            .willReturn(
-                [
-                    .test(id: 0, fullName: "tuist/test-one"),
-                    .test(id: 1, fullName: "tuist/test-two"),
-                ]
-            )
+        try await ServiceContext.withTestingDependencies {
```

Comment:
I checked if it's possible to move this nesting from individual cases, but unfortunately it's not possible because task locals propose this pattern and you can go around it.